openpathsampling/tests/test_snapshot_modifier.py

```diff
@@ -106,10 +118,20 @@ def test_call(self):
                                   new_3D.coordinates)
         assert_array_almost_equal(self.snapshot_3D.velocities,
                                   new_3D.velocities)
-        assert_true(self.snapshot_1D.coordinates is not new_1D.coordinates)
-        assert_true(self.snapshot_1D.velocities is not new_1D.velocities)
-        assert_true(self.snapshot_3D.coordinates is not new_3D.coordinates)
-        assert_true(self.snapshot_3D.velocities is not new_3D.velocities)
+        assert self.snapshot_1D.coordinates is not new_1D.coordinates
+        assert self.snapshot_1D.velocities is not new_1D.velocities
+        assert self.snapshot_3D.coordinates is not new_3D.coordinates
+        assert self.snapshot_3D.velocities is not new_3D.velocities
+        # TODO OPS 2.0: the following tests should probabily work
+        # assert new_1D == self.snapshot_1D
+        # assert new_3D == self.snapshot_3D
```

Comment:
Actually, those probably won't work if we use UUID for snapshot hash/equality as under discussion in #1080. I think we need helper functions (probably only in testing) to check (near) equality of snapshots. In practice, the only reason I can think of to make a copy of snapshot is for testing CV behavior (new UUID means that caching results by UUID doesn't work; forces you to recalculate instead of using cached values.)

Reply:
removed them, I do agree that this might not be wanted behavior